actions/post_actions.jsx

```diff
@@ -225,6 +225,23 @@ export function setEditingPost(postId = '', commentCount = 0, refocusId = '', ti
     };
 }
 
+export function unreadPost(post) {
+    return async (dispatch, getState) => {
+        const state = getState();
+        const channelId = post.channel_id;
+        const timestamp = post.create_at;
+        const userId = getCurrentUserId(state);
+        const data = await dispatch(PostActions.setUnreadPost(userId, post.id));
+        if (data) {
+            dispatch({
+                type: ActionTypes.RECEIVED_POSTS_FOR_CHANNEL_AT_TIME,
```

Comment:
Why is this action needed?

Reply:
not needed, seems to be a leftover from a test trying to understand how to get the separator in the channel. Thanks for spotting!